wrench/src/yaml_helper.rs

```diff
@@ -319,4 +321,53 @@ impl YamlHelper for Yaml {
     fn as_scroll_policy(&self) -> Option<ScrollPolicy> {
         return self.as_str().and_then(|string| string_to_scroll_policy(string))
     }
+
+    fn as_filter_op(&self) -> Option<FilterOp> {
+        if let Some(s) = self.as_str() {
```

Comment:
this function body is overly verbose and could be reduced by some refactoring
not required for this PR though